packages/scene-composer/src/components/three-fiber/AnimationComponent/AnimationComponent.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { Fragment } from 'react';
+import * as THREE from 'three';
+import {  useFrame } from '@react-three/fiber';
+import { GLTF } from 'three/examples/jsm/loaders/GLTFLoader';
+import { useGLTF } from '../ModelRefComponent/GLTFLoader';
+import { useSceneComposerId } from '../../../common/sceneComposerIdContext';
+import { ISceneNodeInternal, useEditorState, useViewOptionState } from '../../../store';
+
+import { IAnimationComponent } from '../../../interfaces';
+
+function pauseAnimation(gltfModel, AnimationMixer, isPaused, AnimationList){
+  try{
+    AnimationList.forEach( function ( clip ) {
+      const animation = THREE.AnimationClip.findByName( gltfModel, clip );
+      AnimationMixer.clipAction(animation).paused = isPaused
+    } );    
+  }
+  catch (error) {
+   // console.error(error);
+  }
+}
+
+function Animate(gltfModel, AnimationMixer, AnimationList){
```

Comment:
nitpick: I might rename this to a verbName like playAnimation to match the other function. Also the class has a lot phrase starting with Animat... so it might stand out more when scanning.